_data/compatibility/casa.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: Casa
+internal_url: /en/compatibility/casa
+logo: /img/compatibility/casa/casa.png
+rbf:
+  tested:
+    date: "2019-10-24"
+    platforms:
+      - iOS
```

Comment:
We have some wallets the have different capabilities on different devices so I think it is best to keep this to one platform. I suppose it is your call on which.

Reply:
ah, OK... I'd go with iOS since it's the one I used for screenshots